client/sites/settings/administration/tools/delete-site/index.jsx

```diff
@@ -271,7 +271,7 @@ class DeleteSite extends Component {
 					) }
 				></NavigationHeader>
 				{ siteId && <QuerySitePurchases siteId={ siteId } /> }
-				<HeaderCakeBack onClick={ this._goBack } />
+				<HeaderCakeBack icon="chevron-left" onClick={ this._goBack } />
```

Comment:
I've fixed this (and moved its position) here: https://github.com/Automattic/wp-calypso/pull/96887/files#diff-69c1c410f1da4882812d44165bcf8688eaf78617e6229d08c77df65c6130e10dR257

Let's remove this as this will create conflict for sure